### PR TITLE
TECH: Add tag variables to modules

### PR DESCRIPTION
**Pull request by smartryanstewart**
Did some formatting as well plus the tags. 

Based on this
[https://smartrent.atlassian.net/jira/software/projects/TECH/boards/5?selectedIssue=TECH-434](url)

TODO:

- [x] Make sure we have all the things tagged
- [x] Figure out a testing method.(Another repo...maybe)

### CODE REVIEW

**Comment by smartryanstewart** [comment]
This is not ready for primetime yet. Just wanting to get some thoughts about where we should test this.

**Comment by rraub** [comment]
lets merge all of our stuff into the `smartrent-main` branch (I just merged in my two PRs with formatting and SSL policy changes into it) So that way all of our diffs wont be mixed up with formatting.

File: modules/ecs/cluster/main.tf [hunk 1]
```diff
@@ -26,13 +26,13 @@ resource "aws_iam_instance_profile" "ecs_instance_profile" {
 # This is created for all load balancers in the cluster, so we can whitelist
 # inbound from these load balancers on the instance security group
 resource "aws_security_group" "lb_security_group" {
-  name = "${aws_ecs_cluster.ecs_cluster.name}-load-balancer"
+  name        = "${aws_ecs_cluster.ecs_cluster.name}-load-balancer"
   description = "${aws_ecs_cluster.ecs_cluster.name} load balancers"
-  vpc_id = var.aws_vpc_id
+  vpc_id      = var.aws_vpc_id
 
   tags = {
```
Comment (rraub) [inline modules/ecs/cluster/main.tf]: We should use including the the tags variable

File: modules/rds/main.tf [hunk 2]
```diff
@@ -6,7 +6,7 @@ resource "aws_security_group" "rds_security_group" {
 
   tags = {
```
Comment (rraub) [inline modules/rds/main.tf]: should be using the tags variable

**Review by rraub — CHANGES_REQUESTED** [review]
Pull in the commits from smartrent-main and hopefully the whitespace diff will go away.

Also we need to actually use these tags variables in the modules (sorry I see the todo now in your PR description 😆 )

**Review by rraub — CHANGES_REQUESTED** [review]
- Need to add tags to the  aws_lb_target_group's, aws_ssm_parameter's, aws_iam_role's, and aws_ecs_service's (there could be a few others too)
- enable propagate_tags  on aws_ecs_service 
Basically anywhere that we are missing tags we should be adding them.

**Comment by smartryanstewart** [comment]
> 
> 
>     * Need to add tags to the  aws_lb_target_group's, aws_ssm_parameter's, aws_iam_role's, and aws_ecs_service's (there could be a few others too)
> 
>     * enable propagate_tags  on aws_ecs_service
>       Basically anywhere that we are missing tags we should be adding them.

So everywhere then lol, guess I misunderstood the module part of the ticket, sorry.

**Review by rraub — APPROVED** [review]
Looks great! Thanks! and sorry the ticket wasn't clear 😬

**Pull request merged**